extensions/interactions/CodeRepl/directives/oppia-interactive-code-repl.component.ts

```diff
@@ -131,8 +131,8 @@ export class InteractiveCodeReplComponent implements
         this.preCode + this.placeholder + this.postCode);
       this.output = '';
     } else {
-      this.code = this.lastAnswer().code;
-      this.output = this.lastAnswer().output;
+      this.code = this.lastAnswer.code;
+      this.output = this.lastAnswer.output;
```

Comment:
Should we add a test that catches this in the future?

Reply:
I am confused. A check for this file or a general check that checks after migration I have nothing left over? This is because this was more of a migration error than a logical error. In AngularJS, we had that `&` syntax which converted it to function that ran in the parents context. A unit test for this won't solve the source of this error as the source is highly unlikely to reoccur in this file (changing the binding from '&' to the equivalent of '<' in Angular. But something that would check this directive migration PR would be really helpful.